src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
Do you think we should use `extends Component {` instead of a function?

> In #2509 I created a version of muiThemeable that behaves more like connect that accepts arguments for mapping a baseTheme to localized style/theme objects (objects that only have the style information that the component needs).

Could you explain the benefit of this approach? I can't see it :grin:.